components/input.css

```diff
@@ -0,0 +1,37 @@
+.form-container{
+    padding: 1rem;
+    margin: 0.5rem;
+    font-size: 1rem;
+    background-color: lightgreen;
```

Comment:
use color variables instead of direct colors

Reply:
ya i think that after all the work of component library done. after that i will refactor